cluster-migration/src/main/java/com/wipro/ats/bdre/clustermigration/beans/MigrationPreprocessorInfo.java

```diff
@@ -0,0 +1,195 @@
+        return srcTable;
+    }
+
+    public void setSrcTable(String srcTable) {
+        this.srcTable = srcTable;
+    }
+
+    public String getFilterCondition() {
+        return filterCondition;
+    }
+
+    public void setFilterCondition(String filterCondition) {
+        this.filterCondition = filterCondition;
+    }
+
+    public String getJtAddress() {
+        return jtAddress;
+    }
+
+    public void setJtAddress(String jtAddress) {
+        this.jtAddress = jtAddress;
+    }
+
+    public String getNnAddress() {
+        return nnAddress;
+    }
+
+    public void setNnAddress(String nnAddress) {
+        this.nnAddress = nnAddress;
+    }
+
+    public String getSrcStgTablePath() {
+        return srcStgTablePath;
+    }
+
+    public void setSrcStgTablePath(String srcStgTablePath) {
+        this.srcStgTablePath = srcStgTablePath;
+    }
+
+    public String getDestStgFolderPath() {
+        return destStgFolderPath;
+    }
+
+    public void setDestStgFolderPath(String destStgFolderPath) {
+        this.destStgFolderPath = destStgFolderPath;
+    }
+
+    public String getDestStgFolderContentPath() {
+        return destStgFolderContentPath;
+    }
+
+    public void setDestStgFolderContentPath(String destStgFolderContentPath) {
+        this.destStgFolderContentPath = destStgFolderContentPath;
+    }
+
+    public String getDestTablePath() {
+        return destTablePath;
+    }
+
+    public void setDestTablePath(String destTablePath) {
+        this.destTablePath = destTablePath;
+    }
+
+    public String getDestTable() {
+        return destTable;
+    }
+
+    public void setDestTable(String destTable) {
+        this.destTable = destTable;
+    }
+
+    public String getDestDb() {
+        return destDb;
+    }
+
+    public void setDestDb(String destDb) {
+        this.destDb = destDb;
+    }
+
+    @PersistableParam(keyName = "source-stg-db")
+    private String srcStgDb;
+    @PersistableParam(keyName = "source-stg-table")
+    private String srcStgTable;
+    @PersistableParam(keyName = "stg-all-part-cols")
+    private String stgAllPartCols;
+    @PersistableParam(keyName = "source-reg-cols")
+    private String srcRegularCols;
+    @PersistableParam(keyName = "source-bp-cols")
+    private String srcBPCols;
+    @PersistableParam(keyName = "source-db")
+    private String srcDb;
+    @PersistableParam(keyName = "source-table")
+    private String srcTable;
+    @PersistableParam(keyName = "filter-condition")
+    private String filterCondition;
+    @PersistableParam(keyName = "job-tracker-address")
+    private String jtAddress;
+    @PersistableParam(keyName = "name-node-address")
+    private String nnAddress;
+    @PersistableParam(keyName = "src-stg-tbl-path")
+    private String srcStgTablePath;
+    @PersistableParam(keyName = "dest-stg-folder-path")
+    private String destStgFolderPath;
+    @PersistableParam(keyName = "dest-stg-folder-content-path")
+    private String destStgFolderContentPath;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)